netstandard/src/netstandard.csproj

```diff
@@ -62,4 +62,41 @@
     <ProjectReference Include="..\..\platforms\$(TargetGroup)\Mono.Android.csproj" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+
+  <PropertyGroup>
+    <CompileDependsOn>
+      GenerateTypeForwards;
+      $(CompileDependsOn);
+    </CompileDependsOn>
+  </PropertyGroup>
+
+  <Target Name="GenerateTypeForwards" DependsOnTargets="ResolveMatchingContract">
```

Comment:
Can you do something like this? (probably move a couple properties to avoid duplication)
```
Inputs="@(ResolvedMatchingContract);$(IntermediateOutputPath)netstandard.forwards.cs;$(MSBuildThisProjectDirectory)GenApi.exclude.$(TargetGroup).txt"
Outputs="$(IntermediateOutputPath)netstandard.forwards.cs"
```